.github/pull.yml

```diff
@@ -0,0 +1,13 @@
+version: "1"
+rules:                      # Array of rules
+  - base: master            # Required. Target branch
```

Comment:
We should decide on the branch we want to keep up to date, and the branch that we want to use as our own master. @deedeeg mentioned that we might want to have a branch named `community` to merge all of our changes there, and keep `master` the same as `upstream/master`.